generator/specbuilder.go

```diff
@@ -414,26 +414,47 @@ func (b *specBuilder) getOrCreateSchemaRef(model reflect.Type) (*openapi3.Schema
 }
 
 func (b *specBuilder) resolveRefPaths() {
-	for _, schemaRef := range b.spec.Model.Components.Schemas {
+	for schemaName, schemaRef := range b.spec.Model.Components.Schemas {
 		// Next make sure the refs point to other schemas, if not already done.
-		for _, propertyRef := range schemaRef.Value.Properties {
+		for propertyName, propertyRef := range schemaRef.Value.Properties {
```

Comment:
I'm getting different outputs on different runs instead of the spec generating being idempotent. I'm noticing we're using maps here, which are iterated randomly. Are we accidentally relying on map order somewhere?